### PR TITLE
s3fs 2022.10.0

### PR DESCRIPTION
**Pull request by SC426**
Jira ticket: https://anaconda.atlassian.net/browse/PKG-154
Upstream repo: https://github.com/fsspec/s3fs
Changelog: https://github.com/fsspec/s3fs/blob/2022.10.0/docs/source/changelog.rst

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -7,12 +7,11 @@ package:
 source:
   fn: s3fs-{{ version }}.tar.gz
   url: https://pypi.io/packages/source/s/s3fs/s3fs-{{ version }}.tar.gz
-  sha256: 6bafc1f6b4e935ea59512c0f38d5cb9c299dbbfe738e40c3d1de8f67b4ee1fd4
+  sha256: e8deb80f20bd0b2059141b874fdb9d6aeb8cce35312ea5f2c02b225a78a00406
 
 build:
   number: 0
   script: {{ PYTHON }} -m pip install . --no-deps -vv
-  noarch: python
```
Comment (skupr-anaconda) [inline recipe/meta.yaml:15]: Skip `py<37`, see https://github.com/fsspec/s3fs/blob/e358d132757da3e47fafaea7420803eed102b19b/setup.py#L34

File: recipe/meta.yaml [hunk 2]
```diff
@@ -22,9 +21,9 @@ requirements:
...
   run:
-    - python >=3.6
-    - aiobotocore ==2.1.0
-    - aiohttp >=3.7.1,<4
+    - python
+    - aiobotocore ==2.4.0
+    - aiohttp !=4.0.0a0,!=4.0.0a1
     - fsspec {{ version }}
 
 test:
```
Comment (skupr-anaconda) [inline recipe/meta.yaml:30]: Please remove `python <3.10` from `test/requires`. It was a temporary fix and it's no more needed.

File: recipe/meta.yaml (hunk 1 above)
Comment (SC426) [reply recipe/meta.yaml:15]: This has been updated

File: recipe/meta.yaml (hunk 2 above)
Comment (SC426) [reply recipe/meta.yaml:30]: This has been removed

**Review by skupr-anaconda — APPROVED** [review]

**Comment by skupr-anaconda** [comment]
Do not forget to remove before merging: 
```
channels:
  scrass_org: s3fs
```

**Review by cbouss — APPROVED** [review]
LGTM

**Pull request merged**